src/styles/scss/mixins.scss

```diff
@@ -20,6 +21,12 @@
   max-width: 31.25rem;
 }
 
+@mixin input-parent-base {
```

Comment:
change to **input-wrapper** or something similar. Needs to have **wrapper** in it.